### ISSUE TITLE
953100 False Positive matching Field cannot be empty.

### ISSUE THREAD

**Issue by EsadCetiner** [coreruleset/coreruleset#3399]
### Description
When you are creating a sieve filter within Roundcube Webmail and you don't supply a filter name, an error message of ``
Field cannot be empty.`` is returned triggering rule 953100. The rule is matching the ``cannot be empty`` https://github.com/coreruleset/coreruleset/blob/71e8dabbad46f00d78cdfaa20dd806cdab6177f1/rules/php-errors.data#L836
Some more specific PHP errors are matched such as the following:
```
Array of object IDs cannot be empty
Filename cannot be empty
Path cannot be empty
cannot be empty when HMAC is requested
individual body cannot be empty
```

I'm not entirely sure if the ``cannot be empty`` phrase should be moved to PL-2 or commented out as in #3390 

### Logs

``
ModSecurity: Warning. Matched "Operator `PmFromFile' with parameter `php-errors.data' against variable `RESPONSE_BODY' (Value: `<!DOCTYPE html>\x0a\x0a<html lang="en" class="iframe">\x0a\x0a<head>\x0a<meta http-equiv="content-ty (49072 characters omitted)' ) [file "/etc/nginx/modsecurity/coreruleset/rules/RESPONSE-953-DATA-LEAKAGES-PHP.conf"] [line "26"] [id "953100"] [rev ""] [msg "PHP Information Leakage"] [data "Matched Data: cannot be empty found within RESPONSE_BODY"] [severity "3"] [ver "OWASP_CRS/4.0.0-rc2"] [maturity "0"] [accuracy "0"] [tag "application-multi"] [tag "language-php"] [tag "platform-multi"] [tag "attack-disclosure"] [tag "paranoia-level/1"] [tag "OWASP_CRS"] [tag "capec/1000/118/116"] [tag "PCI/6.5.6"] [hostname "0.0.0.0"] [uri "/mail/"] [unique_id "170139365471.753794"] [ref "o48346,15v998,48542"]
``

### Your Environment

* CRS version (e.g., v3.3.4): 4.0 RC-2
* Paranoia level setting (e.g. PL1) : 1
* ModSecurity version (e.g., 2.9.6): 3.0.10
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54): Nginx 1.18.0
* Operating System and version: Ubuntu 22.04

### Confirmation

[x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by dune73** [comment]
I'm torn. This indicates a PHP error that the application should catch. But then it's probably so widespread outside a security context that it will hit a lot of users.

Maybe introducing a stricter sibling at PL2 is indeed the way to go. `php-errors.data`  is huge though. Would we just shift items with an opportunistic approach?

**Comment by EsadCetiner** [comment]
After thinking about it for a bit, I've decided that this should be moved to PL-2, there are other more specific errors already matched at PL-1. ``Path cannot be empty`` might also have some false positives but it should be fine.
Opened a PR here: https://github.com/coreruleset/coreruleset/pull/3407
